src/components/TaskCheckbox/TaskCheckbox.tsx

```diff
@@ -10,12 +10,14 @@ const TaskCheckbox: React.FC<TaskCheckboxProps> = ({
   status,
   size = 'default',
   onChange,
+  className,
 }) => {
   const isChecked = status === TaskStatus.Completed;
   const onCheckboxChange = () => onChange && onChange(id);
 
   return (
     <Checkbox
+      className={className}
```

Comment:
Why do we need to change `TaskCheckbox` to add the reminder button?
Passing a `className` as props it kind of a pitfall, I am saying that because we should not be able to change inner styles of the component from outside since component should be atomic. If it's a different type of checkbox it should be a new `type`